schema.sql

```diff
@@ -4,7 +4,7 @@ CREATE TABLE IF NOT EXISTS que_jobs
   run_at      timestamptz NOT NULL DEFAULT now(),
   job_id      bigserial   NOT NULL,
   job_class   text        NOT NULL,
-  args        json        NOT NULL DEFAULT '[]'::json,
+  args        jsonb       NOT NULL DEFAULT '[]'::jsonb,
```

Comment:
The job insert performs the insert as JSON into the JSONB column. Something to do with the postgres transport? Was getting a weird error: `ERROR: unsupported jsonb version number 123` when inserting as JSONB directly. This lets postgres do the marshaling rather than solving it.... plus these payloads aren't that huge in our use case.